<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QComboBox" name="cmb_time_interval">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>15</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>10</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>15</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>30</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>60</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>120</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>300</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>15</y>
     <width>136</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Интервал обновления </string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_run_map_gps">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>445</y>
     <width>85</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Старт</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_stop_map_gps">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>445</y>
     <width>85</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Стоп</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>346</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Текущие значения</string>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Спутниковое время</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ln_date_time">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>80</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Скорость (км.час)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ln_speed">
    <property name="geometry">
     <rect>
      <x>265</x>
      <y>75</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>54</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Широта</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ln_longitude">
    <property name="geometry">
     <rect>
      <x>65</x>
      <y>50</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>145</x>
      <y>50</y>
      <width>54</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Долгота</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ln_latitude">
    <property name="geometry">
     <rect>
      <x>205</x>
      <y>50</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>54</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Азимут</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ln_azimut">
    <property name="geometry">
     <rect>
      <x>65</x>
      <y>80</y>
      <width>56</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Горизонтальная ошибка (м)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ln_erorr">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>110</y>
      <width>66</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>346</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Отображение спутника на карте</string>
   </property>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>20</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Активная карта</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmb_mapviews">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>211</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>60</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Стиль отображения наблюдения</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>245</x>
      <y>55</y>
      <width>46</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="wg_point_style"/>
   </widget>
  </widget>
  <widget class="QGroupBox" name="grbx_save">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>305</y>
     <width>341</width>
     <height>126</height>
    </rect>
   </property>
   <property name="title">
    <string>Сохранение точек</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>20</y>
      <width>76</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>База данных</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ln_path_catalog">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>216</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_select_file">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>15</y>
      <width>26</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>60</y>
      <width>54</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Стиль</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>50</y>
      <width>46</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="wg_style_save"/>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>55</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Сохранять как</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmb_type_obj">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>55</y>
      <width>86</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>точки</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>полилинии</string>
     </property>
    </item>
   </widget>
   <widget class="QCheckBox" name="ch_view_in_map">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>211</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Отображать на активной карте</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pb_save_property">
   <property name="geometry">
    <rect>
     <x>225</x>
     <y>445</y>
     <width>146</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Сохранить настройки</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
